Content/ua_20200713_Feats-2020.js

```diff
@@ -8,4 +8,306 @@ SourceList["UA:F20"] = {
 	group : "Unearthed Arcana",
 	url : "https://media.wizards.com/2020/dnd/downloads/UA2020_Feats.pdf",
 	date : "2020/07/13"
+};
+
+FeatsList["magic initiate [artificer]"] = {
+	name : "Magic Initiate [Artificer]",
+	source : [["UA:F20", 1]],
+	description : "I learn two cantrips and one 1st-level spell of my choice from the artificer's spell list. I can cast the 1st-level spell at its lowest level once per long rest without using a spell slot. Int is my spellcasting ability. I gain proficiency in one artisan's tool that I can use as focus.",
+	spellcastingBonus : [{
+		name : "Artificer cantrip",
+		spellcastingAbility : 4,
+		'class' : 'artificer',
```

Comment:
`class` can be written without the single quotes.

Reply:
The other Magic Initiate feats are written this way. I am more than willing to change it, just want to make sure before breaking from what the current convention is.

Reply:
Hmm. I don't think it's necessary. It might be because class is a reserved word in JS, but I think it ought to work just fine in an object as is. I'd remove the quotes to keep it consistent with the rest of the properties.